### PR TITLE
PHPUnit fixes

### PR DESCRIPTION
**Pull request by dennisameling**
As mentioned in https://github.com/mautic/api-library/issues/205#issuecomment-581047337

### CODE REVIEW

File: tests/Api/ReportsTest.php [hunk 1]
```diff
@@ -38,7 +37,6 @@ public function testGetCustom()
 
         $this->assertErrors($response);
         $this->assertGreaterThanOrEqual(0, $response['totalResults']);
-        $this->assertGreaterThanOrEqual(0, count($response['totalResults']));
```
Comment (escopecz) [inline tests/Api/ReportsTest.php]: Why was this removed? The `totalResutls` param is not in the response anymore?

File: lib/Auth/AbstractAuth.php [hunk 2]
```diff
@@ -235,17 +234,6 @@ protected function createCurlFile($filename, $mimetype = '', $postname = '')
...
 
-    /**
-     * @param $isPost
-     * @param $parameters
-     *
-     * @return array
-     */
-    protected function getQueryParameters($isPost, $parameters)
-    {
-        return ($isPost) ? array() : $parameters;
```
Comment (escopecz) [inline lib/Auth/AbstractAuth.php]: @alanhartless fixed this on the 3.0 branch like this: https://github.com/mautic/api-library/commit/44a4db754768da71cdbed0b0a1cb9297ecff310d

File: tests/Api/ReportsTest.php (hunk 1 above)
Comment (dennisameling) [reply tests/Api/ReportsTest.php]: It's still there, but in the line above it, it's already checked whether its value is greater than or equal to 0. Since it's an integer, it's throwing a warning in PHP 7.2+ as integers are non-countable (https://www.php.net/manual/en/migration72.incompatible.php#migration72.incompatible.warn-on-non-countable-types)

File: lib/Auth/AbstractAuth.php (hunk 2 above)
Comment (dennisameling) [reply lib/Auth/AbstractAuth.php]: Fixed :+1:
